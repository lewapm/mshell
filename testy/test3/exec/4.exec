
$shell < $inf 2> $errf | sort > $outf

