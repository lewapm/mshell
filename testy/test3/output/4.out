chdir to pm
chdir to servers
done
fs
fs
init
init
Makefile
Makefile
pm
pm
rs
rs
servers
teststr.tar.gz
