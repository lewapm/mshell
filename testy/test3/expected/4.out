Makefile
Makefile
chdir to pm
chdir to servers
done
fs
fs
init
init
pm
pm
rs
rs
servers
teststr.tar.gz
